src/build/build_helpers.jl

```diff
@@ -211,11 +217,11 @@ function get_tax_revenue_for_consumer(jm, m, cr::ConsumerRef)
             for tax in output.taxes
                 if cr.subindex === nothing
                     if get_full(tax.agent) == get_full(cr)    
-                        push!(taxes, :($(tax.rate) * $(jm[get_comp_supply_name(output)]) * $(output.commodity) * $(pf.sector)))
+                        push!(taxes, eval(swap_our_param_with_val((tax.rate))) * (jm[get_comp_supply_name(output)]) * tojump(jm,output.commodity) * tojump(jm,pf.sector))
```

Comment:
Same thing as above, all these `eval(swap...` things are not correct, we should just call `tojump` on the `tax.rate`.